galata/test/documentation/commands.test.ts-snapshots/commandsList-documentation-linux.json

```
@@ -177,7 +177,7 @@
   },
   {
     "id": "apputils:display-shortcuts",
-    "label": "Show Keyboard Shortcuts",
+    "label": "Show Keyboard Shortcuts...",
```

This looks to have been carried over from another PR.

```suggestion
    "label": "Show Keyboard Shortcuts",
```